xbmc/windowing/gbm/DRMUtils.cpp

```diff
@@ -425,15 +425,12 @@ bool CDRMUtils::FindPlanes()
 
   m_primary_plane->plane = FindPlane(plane_resources, m_crtc_index, KODI_VIDEO_PLANE);
   m_overlay_plane->plane = FindPlane(plane_resources, m_crtc_index, KODI_GUI_10_PLANE);
-  m_overlay_plane->format = DRM_FORMAT_XRGB2101010;
-  m_overlay_plane->fallbackFormat = DRM_FORMAT_XRGB8888;
 
   /* fallback to 8bit plane if 10bit plane doesn't exist */
   if (m_overlay_plane->plane == nullptr)
   {
     drmModeFreePlane(m_overlay_plane->plane);
     m_overlay_plane->plane = FindPlane(plane_resources, m_crtc_index, KODI_GUI_PLANE);
-    m_overlay_plane->format = DRM_FORMAT_XRGB8888;
```

Comment:
I am wondering if we need to set `useFallbackFormat=true` here after this change, I think that RK3288/RK3399 supports 10-bit gpu surface but there is no drm plane that supports 10-bit rgb, I suspect there could be an issue with egl config in such case, will get back with test results

Reply:
@Kwiboo the `useFallbackFormat` was added because some intel hardware exposes 10bit planes but doesn't have a 10bit EGL mode so the `chooseconfig()` fails. Maybe there is a more elegant way though.

Reply:
I think the format and fallback usage makes much more sense with the changes in this PR, thanks for that 👍

I am thinking that RK3288/RK3399 and other HW may have similar but reversed issue, where it may pick a 10bit EGL config yet don't have a 10-bit plane (`format` defaults to `XRGB2101010` for 8-bit planes with this line dropped), still investigating but I think keeping this line or setting `useFallbackFormat=true` here should make it choose a 8-bit config.

Reply:
Adding `m_overlay_plane->SetFormat(DRM_FORMAT_XRGB8888);` here probably makes most sense and won't make `useFallbackFormat` having two different meanings.

Reply:
I think that's a good idea too